types/protobuf.go

```diff
@@ -101,17 +101,17 @@ func (tm2pb) ValidatorUpdate(val *Validator) abci.ValidatorUpdate {
 // TODO: add cases when new pubkey types are added to crypto
 func (tm2pb) PubKey(pubKey crypto.PubKey) abci.PubKey {
```

Comment:
I'll remove these two functions later on as this pr is already growing large and this would cause it to grow greatly